app/controllers/email/CaptureEmailController.scala

```diff
@@ -100,4 +102,78 @@ class CaptureEmailController @Inject()(val vatSubscriptionService: VatSubscripti
       case (None, _) => Future.successful(errorHandler.showInternalServerError)
     }
   }
+
+  def showPrefJourney: Action[AnyContent] = contactPreferencePredicate.async { implicit user =>
+
+    if (appConfig.features.letterToConfirmedEmailEnabled()){
+      val validationEmail: Future[Option[String]] = user.session.get(SessionKeys.validationEmailKey) match {
+        case Some(email) => Future.successful(Some(email))
+        case _ => vatSubscriptionService.getCustomerInfo(user.vrn) map {
+
+          case Right(details) => Some(details.ppob.contactDetails.flatMap(_.emailAddress).getOrElse(""))
+          case _ => None
+        }
+      }
+
+      val prepopulationEmail: Future[String] = validationEmail map { validation =>
+        user.session.get(SessionKeys.prepopulationEmailKey)
+          .getOrElse(validation.getOrElse(""))
+      }
+
+      for {
+        validation    <- validationEmail
+        prepopulation <- prepopulationEmail
+
+      } yield {
+        validation match {
+          case Some(valEmail) =>
+            Ok(captureEmailView(emailForm(valEmail).fill(prepopulation),
+              emailNotChangedError = false, valEmail,controllers.email.routes.CaptureEmailController.submitPrefJourney(),false))
```

Comment:
Shouldn't the new boolean be `true` here? You want the back link to go back to the "add email" page in this journey.